airbyte/_executors/util.py

```
@@ -236,6 +240,12 @@ def get_connector_executor(  # noqa: PLR0912, PLR0913, PLR0915 # Too many branch
         if use_host_network is True:
             docker_cmd.extend(["--network", "host"])
 
+        if runas_host_user is True:
+            try:
+                docker_cmd.extend(["--user", f"{os.getuid()}:{os.getgid()}"])
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)